web/source/keymanweb.js

```diff
@@ -1021,26 +1021,47 @@
    * Scope        Private
    * Description  Save list of inputs for non-touch devices (desktop browsers)
    */       
-  keymanweb.setupDesktopPage = function()
-  { 
-    for(var k=0; k<2; k++)
-    {
+  keymanweb.setupDesktopPage = function() { 
+    for(var k=0; k<2; k++) {
```

Comment:
keymanweb.util doesn't exist yet.  What would be the best code file to place it within?  Create a new kmwutil.js file?